extensions/image-preview/src/preview.ts

```diff
@@ -223,15 +223,15 @@ class Preview extends Disposable {
 	private getResourcePath(webviewEditor: vscode.WebviewPanel, resource: vscode.Uri, version: string) {
 		switch (resource.scheme) {
 			case 'data':
```

Comment:
What user operation or situation would run codes in the `case 'data'`?
I guess that the `case 'git'` part would be fine. Not sure about the `case 'data'` section.